apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/AbstractClassEnhancePluginDefine.java

```diff
@@ -64,12 +67,21 @@
         String[] witnessClasses = witnessClasses();
         if (witnessClasses != null) {
             for (String witnessClass : witnessClasses) {
-                if (!WitnessClassFinder.INSTANCE.exist(witnessClass, classLoader)) {
+                if (!WitnessFinder.exist(witnessClass, classLoader)) {
```

Comment:
I was not saying change this to static method. My meaning is you could get `var = WitnessFinder.INSTANCE before all loops.